pipeline_dp/accumulator.py

```diff
@@ -0,0 +1,117 @@
+import abc
+import typing
+import pickle
+from functools import reduce
+
+
+def merge(accumulators: typing.Iterable[
+  'Accumulator'] = None) -> 'Accumulator':
+  """Merges the accumulators.
+
+  Args:
```

Comment:
Nit: the short description is enoughm remove starting from "Args:...

Reply:
Done